tests/test_helpers.py

```diff
@@ -144,6 +144,41 @@
      "    aka:\n"
      "        -A. B. Friend\n        -AB Friend\n        -Tony Friend\n"
      ),
+    (["helper", "l_abstract", "run", "-a", "scopatz”, "-y", "2018", "-l", ”upton", "-t", "tape"],
+     "---------------------------------------\n"
+     "Authors: - scopatz - afriend\n"
+     "Meeting to check flexibility on dates\n"
+     "columbiau, apam\n"
+     "Location: Upton NY\n"
+     "Start: 5-22-2018\n"
+     "Type: award, Status: accepted\n"
+     "Project: 18sob_clustermining\n"
+     "Title: We pulled apart graphite with tape\n"
+     "Abstract: Graphitic Dephenestration"
```

Comment:
Let's make it look like this in the output:
```
Title:   We pulled apart graphite with tape

Anthony B. Scopatz,  Anthony F. Friend

Abstract: Graphitic Dephenestration
```
to get the names you need to dereference from the people/contacts collections. There is a function in tools that does that `get_person_contact()`.  look in preslistbuilder to see how to use it.